server-sdk/src/main/java/com/amazonaws/ml/mms/servingsdk/http/Request.java

```diff
@@ -0,0 +1,18 @@
+package com.amazonaws.ml.mms.servingsdk.http;
+
+import java.util.List;
+import java.util.Map;
+
+public interface Request {
+    List<String> getHeaderNames();
+    String getRequestURL();
+    String getRequestURI();
+    String getRequestPath();
+    Map<String, List<String>> getParameterMap();
+    List<String> getParameter(String k);
+    String getContentType();
+    String getRequestedSessionId();
```

Comment:
We don't support session

Reply:
True. It was for later implementation as well. I will remove it for now.